src/main/java/tasker/tasker/configuration/HttpClientConfig.java

```diff
@@ -0,0 +1,15 @@
+package tasker.tasker.configuration;
+
+import org.springframework.boot.web.client.RestTemplateBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.client.RestTemplate;
+
+@Configuration
```

Comment:
pls check configuration properties 
pls refer to the https://www.baeldung.com/configuration-properties-in-spring-boot

Reply:
Added ConfigProperties